.github/workflows/ci.yml

```
@@ -56,5 +57,84 @@ jobs:
 
       - run: mix test --warnings-as-errors --cover
 
+  dialyzer:
+    runs-on: ubuntu-22.04
+    env:
+      MIX_ENV: test
```

Did you want this to be `MIX_ENV: dev`?

```suggestion
      MIX_ENV: dev
```